packages/paste-core/components/combobox/__tests__/virtualization.spec.tsx

```diff
@@ -161,19 +160,19 @@ describe('Combobox - Virtualization', () => {
       expect(first).toHaveAttribute('aria-setsize', '5');
       expect(first).toHaveAttribute('aria-posinset', '1');
 
-      expect(second).toHaveAttribute('transform', 'translateY(8px)');
+      expect(second).toHaveAttribute('transform', 'translateY(44px)');
       expect(second).toHaveAttribute('aria-setsize', '5');
       expect(second).toHaveAttribute('aria-posinset', '2');
 
-      expect(third).toHaveAttribute('transform', 'translateY(44px)');
+      expect(third).toHaveAttribute('transform', 'translateY(80px)');
       expect(third).toHaveAttribute('aria-setsize', '5');
       expect(third).toHaveAttribute('aria-posinset', '3');
 
-      expect(fourth).toHaveAttribute('transform', 'translateY(80px)');
+      expect(fourth).toHaveAttribute('transform', 'translateY(116px)');
       expect(fourth).toHaveAttribute('aria-setsize', '5');
       expect(fourth).toHaveAttribute('aria-posinset', '4');
 
-      expect(fifth).toHaveAttribute('transform', 'translateY(116px)');
+      expect(fifth).toHaveAttribute('transform', 'translateY(152px)');
```

Comment:
Did we ever determine the cause of these calculation differences?

Reply:
So it's basically a corrected test result.

Reply:
Yeah, I think these are the correct values, they also match the values rendered in storybook.